src/actions/blockActions.ts

```diff
@@ -131,17 +133,29 @@ export function fetchBlocks(page = 1, chain?: string) {
     try {
       dispatch(requestBlocks(page))
 
-      const neo2 = await (
-        await fetch(`${GENERATE_BASE_URL('neo2', false)}/blocks/${page}`)
-      ).json()
-
-      const neo3 = await (
-        await fetch(`${GENERATE_BASE_URL('neo3', false)}/blocks/${page}`)
-      ).json()
-
-      const all = sortedByDate(neo2.items, neo3.items)
-
-      dispatch(requestBlocksSuccess(page, { neo2, neo3, all }))
+      const options = [
```

Comment:
I see this constant referenced in a few files... Would be better to move it to `constants.ts` and import it for easier maintainability/readability imo

Reply:
resolved